components/Events/FAQ.tsx

```diff
@@ -0,0 +1,46 @@
+'use client'
+
+import { eventsFaqData } from '@/lib/data';
+import Image from 'next/image';
+import React, { useState } from 'react'
```

Comment:
no need to import `React` -> cause larger bundle size

```suggestion
import { useState } from 'react'
```